src/constants/constants.js

```diff
@@ -10,9 +10,15 @@ export const CUSTOM_TYPES_BUTTONS = {
   standard: 'ui button'
 };
 
+export const LIKES = {
+  MAX: 15,
+  MIN: 0
+};
+
 export const STYLE = {
   BUTTON: {
     standard: 'circular black ui icon button',
+    disabled: 'circular black ui icon button disabled',
```

Comment:
There are a couple of keywords that are repeated multiple times in this constant. You can add the common ones directly on the button as className and only have here the ones that extraClasses. And in the Button code you can do something like this:
<Button className={`circular black ui icon ${extraClass}`)></Button>
And extraClass will contain anything extra you need like 'disabled' or 'disabled olive anyClassHere'